.github/labeler.yml

```diff
@@ -0,0 +1,16 @@
+ci:
+- changed-files:
+  - any-glob-to-any-file:
+    - .github/**
+
+dependencies:
+- changed-files:
+  - any-glob-to-any-file:
+    - .github/worklflows/requirements/**
```

Comment:
same question as https://github.com/spack/spack-gantry/pull/22#discussion_r1535082134 and just a small typo with `workflows`

Reply:
Good eye! Fixed!